assets/css/ansi.css

```diff
@@ -14,7 +14,7 @@ to be consistent with the editor.
   --ansi-color-magenta: #c678dd;
   --ansi-color-cyan: #56b6c2;
   --ansi-color-white: white;
-  --ansi-color-light-black: #5c6370;
+  --ansi-color-light-black: #737a8c;
```

Comment:
This font is used on white background I think. Are we sure we want to change it?

Reply:
The difference on white backgrounds is less prominent than on dark backgrounds, but the old gray is no doubt better, and I'm keeping it.

<img width="935" alt="Screen Shot 2022-01-13 at 13 49 03" src="https://user-images.githubusercontent.com/5660941/149374768-735b68cf-2c39-4f5a-84d0-07d36269471d.png">
<img width="923" alt="Screen Shot 2022-01-13 at 13 49 33" src="https://user-images.githubusercontent.com/5660941/149374773-644bbce6-69ef-43c3-88ae-0e3e118e5e08.png">